shell/browser/api/atom_api_app.cc

```diff
@@ -846,11 +846,11 @@ void App::SetAppPath(const base::FilePath& app_path) {
 }
 
 #if !defined(OS_MACOSX)
-void App::SetAppLogsPath(base::Optional<base::FilePath> custom_path,
-                         mate::Arguments* args) {
+void App::SetAppLogsPath(util::ErrorThrower thrower,
```

Comment:
Why reverse the order of the args here?  It seems more logicall that the first arg would be the path.

Reply:
fair! i'll change this 🙇‍♀

Reply:
Currently we have 2 patterns for passing args:

1. `Func(v8::Isolate* isolate, arg)`: which follows convention of V8 and Node.js
2. `Func(arg, mate::Arguments* args)`: as `args` acts as rest arguments in js

Since `ErrorThrower` is a thin wrapper of `v8::Isolate`, I think it makes sense to put it as first arg.